chroma-manager/Makefile

```diff
@@ -267,6 +269,7 @@ download: requirements
 
 docs: requirements version
 	set -e ; \
+	export PYTHONPATH=$$(pwd)/../chroma-common:$$PYTHONPATH; \
```

Comment:
Acknowledging that the line continuators of the following lines were not lined up already, let's take this opportunity to line them up.